README.md

```diff
@@ -59,7 +59,11 @@ app.get(
     // The node_modules/ is always included.
     includePaths: ['./client/'],
 
-    // defaults to false
+    // Change the output format of of the final CSS file.
+    // Defaults to nested. Values: nested, expanded, compact, compressed.
```

Comment:
Can you change this line of comment to:

    Defaults to "nested". Can be one of: "nested", "expanded", "compact", or "compressed".